plugins/CoreHome/angularjs/common/services/piwik.js

```diff
@@ -61,6 +65,22 @@
                 return false;
             }
         }
+
+        function updateDateInTitle( date, period ) {
+            piwikApi = piwikApi || $injector.get('piwikApi');
+            piwikApi.fetch(
```

Comment:
I wonder if we could keep it simple and if the date was changed in JS, simply show the same date format in the title as the date picker does? It wouldn't be hugely formatted, but maybe good enough.

Reply:
It would simplify a few things in this PR for sure. `Controller::getLongPrettyDate()` would not be needed (I think), so all changes to `core/plugin/Controller.php` and `plugins/CoreHome/Controller.php` could be reverted.

Reply:
A non-essential XHR would also be avoided (tiny perf gain for the front-end app)

Reply:
@spacenate Sounds good, would be great if you can make these changes :+1: